### PR TITLE
Making  note for administrative use field optional

### LINKED ISSUE #3077: CiviForm admin UX: allow admin to leave the program description field blank
**Is your feature request related to a problem? Please describe.**
A CiviForm admin cannot leave the admin program description field blank 

**Describe the solution you'd like**
Make the admin program description field optional

**Additional context**
From Admin UX feedback sheet

**Screenshot**

Screen Shot 2022-08-05 at 2 20 34 PM

### PR DESCRIPTION
**Pull request by azhangggggg**
### Description

This PR makes program description field optional

## Release notes

The title of the pull request will be used as the default release notes description. If more detail is needed to communicate to partners the scope of the PR's changes, use this release notes section.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers).
- [x] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [ ] Extended the README / documentation, if necessary

#### User visible changes

- [ ] Followed steps to [internationalize new strings](https://github.com/civiform/civiform/wiki/Internationalization-%28i18n%29#internationalization-for-application-strings)
- [ ] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [ ] Tested on mobile view. See [mobile device mode](https://developer.chrome.com/docs/devtools/device-mode/)
- [x] Manually tested at 200% size
- [x] Manually evaluated tab order

#### New Features

- [ ] Add new FeatureFlag env vars to `server/conf/helper/feature-flags.conf`
- [ ] Conditioned new functionality on a [FeatureFlag](https://github.com/civiform/civiform/wiki/Feature-Flags)
- [ ] Wrote browser tests with the feature flag off and on, etc.

### Instructions for manual testing

If instructions are needed for manual testing by reviewers, include them here.

### Issue(s) this completes

Fixes #3077

### CODE REVIEW

**Review by bion — CHANGES_REQUESTED** [review]
server side validations need updated

File: server/test/services/program/ProgramServiceTest.java [hunk 1]
```diff
@@ -485,22 +475,6 @@ public void validateProgramDataForCreate_requiresTIListInSelectTiMode() {
             CiviFormError.of("One or more TI Org must be selected for program visibility"));
   }
 
-  @Test
```
Comment (azhangggggg) [inline server/test/services/program/ProgramServiceTest.java:488]: this test seems to be duplicate with the one above it so I deleted this one. lmk if it was intended tho, I can add it back.

Comment (bion) [reply server/test/services/program/ProgramServiceTest.java:488]: The method under test is different, `validateProgramDataForCreate` vs `validateProgramDataForUpdate`

**Review by bion — CHANGES_REQUESTED** [review]

File: server/test/services/program/ProgramServiceTest.java (hunk 1 above)
Comment (azhangggggg) [reply server/test/services/program/ProgramServiceTest.java]: sorry my bad, added the `validateProgramDataForUpdate` test back.

**Review by bion — APPROVED** [review]

**Pull request merged**